guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -65,6 +65,12 @@ endif::[]
 # ls __my_file_repo__
 PULP_MANIFEST test.txt
 ----
+. Configure additional import paths to sync a file repository that exists on the {Project}. For example, if a file repository is at `/root/myrepo`, enter:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --foreman-proxt-content-pulpcore-additional-import-paths __/root/myrepo__
```

Comment:
`/root/something` is a bad location because the root user can access it and the application can never access it.

Reply:
> `/root/something` is a bad location because the root user can access it and the application can never access it.

Can you suggest alternate location? Thanks!

Reply:
My first thought was to keep it vague and say `__/path/to/myrepo__`. I do wonder if SELinux labels also limit the import path, but I don't have time to dive into it now.